cmd/logger.go

```diff
@@ -137,6 +155,24 @@ func getSource() string {
 	return fmt.Sprintf("[%s:%d:%s()]", filename, lineNum, funcName)
 }
 
+func printJSON(level, source, cause, msg string, stack []string, data ...interface{}) {
+	log := &logEntry{
```

Comment:
No need to write this as a separate line, you can directly do `json.Marshal(&logEntry{})`